sys/kern/src/arch/rv64/interrupts.rs

```diff
@@ -0,0 +1,46 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Unable to post a comment on the commit message, so posting here. riscv-rt crate sets up trap table using direct mode. I don't think we benefit much from the vectored mode here as it is just copying the same entrypoint to all interrupts.. What did not work without this change and do you see difference in behavior with this change? RV64 on m-mode seems to handle traps just fine as it is set up currently.